tests/func-tests/defaults_test.go

```diff
@@ -212,7 +212,7 @@ var _ = Describe("Check Default values", Label("defaults"), Serial, func() {
 
 	Context("virtualMachineOptions defaults", func() {
```

Comment:
The defaults test fails now. Please remove this test (the whole `Context("virtualMachineOptions defaults")`)

There is no point to check it. If you like, you can add a test to make sure that `virtualMachineOptions` is `nil`.

Reply:
Done.